### PR TITLE
Hotfix new testnet

### PR DESCRIPTION
**Pull request by MatusKysel**

### CODE REVIEW

File: for_devs/local-net [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/env /usr/local/bin/python3
```
Comment (JakubFornadel) [inline for_devs/local-net:1]: this should not have been changed probably ?

Comment (rattrap) [reply for_devs/local-net:1]: We want to use the python version from `venv` so this is incorrect

**Review by rattrap — CHANGES_REQUESTED** [review]

File: for_devs/local-net (hunk 1 above)
Comment (kstdl) [reply for_devs/local-net:1]: Sorry, missed that. That should be only my local change

**Comment by justinsnapp** [comment]
Forced push this branch to testnet.

**Pull request closed without merging**